src/librustdoc/html/render/context.rs

```diff
@@ -112,6 +112,10 @@ pub(crate) struct SharedContext<'tcx> {
     /// Storage for the errors produced while generating documentation so they
     /// can be printed together at the end.
     errors: Receiver<String>,
+    /// Document items that have lower than `pub` visibility.
```

Comment:
`SharedContext` is accessible from `DocContext` which contains this information into its field `render_options`. Normally you shouldn't need to add these two fields here.